src/components/user/edit/password.vue

```diff
@@ -32,9 +33,7 @@ except according to the terms contained in the LICENSE file.
           {{ $t('action.change') }} <spinner :state="awaitingResponse"/>
         </button>
       </form>
-      <template v-else>
-        {{ $t('cannotChange') }}
-      </template>
+      <p v-else>{{ $t('cannotChange') }}</p>
```

Comment:
what's the significance of changing from `<template>` to `<p>`?

Reply:
Not a ton. We generally use `<p>` in a `.panel-body`, because there are styles on `<p>`, for example, for `max-width`. I'm not sure those styles end up mattering here, but `<p>` certainly doesn't hurt. I used `<p>` for the OIDC case since I think it's a little better, then wanted to be consistent and changed `<template>` to `<p>` in this case as well.